utils/objectUtils.js

```diff
@@ -4,7 +4,7 @@ const entityType = { SUITE: 'SUITE', TEST: 'STEP' },
     getStartLaunchObject = (options = {}) => ({
         launch: process.env.RP_LAUNCH_NAME || options.launchname || 'Unit Tests',
         tags: options.tags,
-        start_time: new Date().valueOf()
+        startTime: new Date().valueOf()
```

Comment:
It is strange. Looks like in RP v5 reporting compatibility with v4 is broken. I believed RP team has fixed this.@DzmitryHumianiuk ?
Also this changes have to be in rp-js client to fix other integration.

Reply:
RPv5 api will accept both `start_time` and `startime` attribute.
as a backward compatibility.

@pbortnik and @ihar-kahadouski can share details.

Reply:
@DzmitryHumianiuk yes it should. But seems it doesn't work if it is required to make such changes in reporter